src/modules/imageresizer/ui/Views/InputPage.xaml

```diff
@@ -6,62 +6,53 @@
     xmlns:p="clr-namespace:ImageResizer.Properties"
     xmlns:ui="http://schemas.lepo.co/wpfui/2022/xaml">
 
-    <Grid>
+    <Grid Name="RootGrid">
         <Grid.RowDefinitions>
             <RowDefinition Height="Auto" />
-            <RowDefinition Height="Auto" />
+            <!--  ComboBox  -->
             <RowDefinition Height="*" />
+            <!--  other controls  -->
         </Grid.RowDefinitions>
 
         <ComboBox
             x:Name="SizeComboBox"
-            Grid.Row="1"
+            Grid.Row="0"
             Height="64"
-            Margin="16,16,16,24"
+            Margin="16"
             HorizontalAlignment="Stretch"
             VerticalAlignment="Stretch"
             VerticalContentAlignment="Stretch"
             ItemsSource="{Binding Settings.AllSizes}"
             SelectedIndex="{Binding Settings.SelectedSizeIndex}">
             <ComboBox.Resources>
                 <DataTemplate DataType="{x:Type m:ResizeSize}">
-                    <Grid
-                        Margin="2,0,0,0"
-                        VerticalAlignment="Center"
-                        AutomationProperties.Name="{Binding Name}">
+                    <Grid AutomationProperties.Name="{Binding Name}">
```

Comment:
![image](https://github.com/microsoft/PowerToys/assets/9866362/23fc12ab-eaa9-41be-b9e0-49b91ff52e19)

Text should be center aligned in the combobox. Top margin on the grid might fix that.